src/serac/physics/base_physics.cpp

```diff
@@ -98,14 +100,17 @@ void BasePhysics::initializeOutput(const serac::OutputType output_type, const st
   }
 }
 
-void BasePhysics::outputState() const
+void BasePhysics::outputState(const std::string output_directory) const
 {
   switch (output_type_) {
     case serac::OutputType::VisIt:
       [[fallthrough]];
     case serac::OutputType::ParaView:
       dc_->SetCycle(cycle_);
       dc_->SetTime(time_);
+      if (!output_directory.empty()) {
+        dc_->SetPrefixPath(output_directory);
```

Comment:
Is this needed every output cycle? I think it would be nice if we could give this directory string in the initialize step and then we don't have to carry it around for every output.

Reply:
done.